src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/UserControllerTest.java

```diff
@@ -268,55 +268,6 @@ void testActivateModifyUserWithSuperAdmin() throws Exception {
         Assertions.assertEquals(workflowEntityBefore.size(), workflowEntityAfter.size());
     }
 
-    @Test
-    void testActivateModifyUserWithSuperAdminAndFailWithNoFullName() throws Exception {
```

Comment:
Why was this removed? 
We still have fullName in the db?

Reply:
The test case involves an attempt to activate a `user_account` who has a null `user_full_name`. The changes in this PR add a `NOT NULL` constraint to that field, so this is no longer a possible scenario.

This comment has prompted me to revise the `UserActivateValidator` class which checked for this condition.